docs/aquacomputer_d5next.rst

```diff
@@ -100,6 +100,16 @@ pwm_enable values are:
 [4-11] Follow fan[1-8], if available and device supports
 ====== ==========================================================
 
+Possible pwm_enable values on Aquaero are:
```

Comment:
Add Aquaero to the sentence above the previous table, and note that that table applies only for those devices.